Apps/WebClient/src/ClientApp/src/components/report/immunizationHistory.vue

```diff
@@ -250,8 +255,58 @@ export default class ImmunizationHistoryReportComponent extends Vue {
                             </b-col>
                         </b-row>
                     </b-col>
-                    <b-col data-testid="immunizationItemStatus" class="col-2">
-                        {{ immzRecord.status }}
+                </b-row>
+                <b-row>
+                    <b-col>
+                        <h4>Recommended Immunizations</h4>
+                    </b-col>
+                </b-row>
+                <b-row>
+                    <b-col>
+                        <div>
+                            DISCLAIMER: Provincial Immunization Registry record
+                            only. Immunization history displayed may not portray
+                            the client’s complete immunization history and may
+                            impact forecasted vaccines. For information on
+                            recommended immunizations, please visit
+                            <a>http://www.immunizebc.ca</a> or contact your
```

Comment:
@ninosamson can you please confirm the text and the url here as I'm seeing http vs https in the desc. of the 9582 vs 9889

Reply:
updated to https